staging/src/k8s.io/apiserver/pkg/authorization/authorizer/interfaces.go

```diff
@@ -157,3 +157,13 @@ const (
 	// to allow or deny an action.
 	DecisionNoOpinion
 )
+
+const (
+	MechanismAnnotationKey = "authorization.k8s.io/mechanism"
```

Comment:
This will need documenting - see https://kubernetes.io/docs/reference/labels-annotations-taints/audit-annotations/